### PR TITLE
Redesigned the home page and modified the Basic file structure.

### PR DESCRIPTION
**Pull request by ghost**
I wrote HTML & CSS from scratch and also updated the file structure a bit.

coder's community
.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body>
+    <!-- Container Starts -->
+    <div class="container">
+      <!-- Navigation Starts -->
+      <nav>
+
+        <div class="logo">
+          <img class="logo-img" src="/media/png/unnamed.png" alt="" />
```
Comment (rohan-kulkarni-25) [inline home.html]: - IN src you need to add `./` or just don't add anything then only this will work when hosted on github pages

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: home.html (hunk 1 above)
Comment (ghost) [reply home.html]: Should I remove `../` and `/` from CSS file too?

```
/* Variables and Font-Faces */
@font-face {
  font-family: "Druk Wide Bold";
  src: url(../media/font/Druk\ Wide\ Bold.ttf);
}
@font-face {
  font-family: "Work Sans";
  src: url("../media/font/WorkSans-Regular.ttf"),
    url("../media/font/WorkSans-SemiBold.ttf");
}

```

Comment (rohan-kulkarni-25) [reply home.html]: `../` will not make a issue but just `/` without a dot is not supported in many cases

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: home.html (hunk 1 above)
Comment (ghost) [reply home.html]: okayy resolve this issue right now...

**Review by Subhradeep10 — COMMENTED** [review]
Can I know the source of the images?

**Comment by ghost** [comment]
Sources (links):
main illustration - https://drawkit.io/product/project-product-manager-illustrations

**Pull request merged**